lib/macho/headers.rb

```diff
@@ -50,6 +56,8 @@ module MachO
     CPU_TYPE_ANY => "CPU_TYPE_ANY",
     CPU_TYPE_I386 => "CPU_TYPE_I386",
     CPU_TYPE_X86_64 => "CPU_TYPE_X86_64",
+    CPU_TYPE_ARM => "CPU_TYPE_ARM",
+    CPU_TYPE_ARM64 => "CPU_TYPE_ARM64",
```

Comment:
I'm not sure how you feel about this, especially since this has already been exposed publicly, but I'd like us to consider changing the right hand side from a string that is identical to the constant name (both ugly and unwieldy when exposed to a user or in the `ruby-macho` API) to a shorter symbol. Again, I'm proposing to use the established identifiers from `man 3 arch`.